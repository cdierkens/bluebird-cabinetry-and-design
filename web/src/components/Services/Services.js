import React from "react";
import Service from "../Service";
import styles from "./Services.module.css";

const Services = () => {
  return (
    <div className={styles.ServicesContainer}>
      <Service />
      <h2 className={styles.ServicesH2}>Services</h2>
    </div>
  );
};

export default Services;
