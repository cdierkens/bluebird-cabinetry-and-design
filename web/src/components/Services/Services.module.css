.ServicesContainer {
  @apply bg-gray-light;
}

.ServicesGrid {
  @apply pt-8;
}

.IconContainer {
  @apply flex flex-col items-center justify-start text-center col-span-3;
}

.FirstIconContainer {
  @apply flex flex-col items-center justify-start text-center col-span-3;
}

@screen sm {
  .IconContainer,
  .FirstIconContainer {
    @apply col-span-2;
  }
}

@screen md {
  .FirstIconContainer,
  .IconContainer {
    @apply col-span-1;
  }

  .FirstIconContainer {
    @apply col-start-2;
  }
}
