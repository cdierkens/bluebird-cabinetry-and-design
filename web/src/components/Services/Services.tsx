import React from "react";
import {
  ConstructionIcon,
  DesignIcon,
  LayoutIcon,
  MaterialIcon,
  RemodelIcon,
  RenderingIcon,
} from "../../icons";
import Button from "../Button";
import Container from "../Container";
import * as styles from "./Services.module.css";

const Services: React.FC = () => {
  return (
    <div className={styles.ServicesContainer}>
      <Container className={styles.ServicesGrid}>
        <div className="grid grid-cols-2 sm:grid-cols-6">
          <div className={styles.IconContainer}>
            <DesignIcon className="h-12 mb-2 text-blue-dark" />
            <span>
              Cabinetry Design <br />
              &amp; Sales
            </span>
          </div>

          <div className={styles.IconContainer}>
            <RemodelIcon className="h-12 mb-2 text-blue-dark" />
            <span>Remodeling</span>
          </div>

          <div className={styles.IconContainer}>
            <ConstructionIcon className="h-12 mb-2 text-blue-dark" />
            <span>New Construction</span>
          </div>

          <div className={styles.IconContainer}>
            <RenderingIcon className="h-12 mb-2 text-blue-dark" />
            <span>Renderings</span>
          </div>

          <div className={styles.IconContainer}>
            <MaterialIcon className="h-12 mb-2 text-blue-dark" />
            <span>Material &amp; Appliance Selections</span>
          </div>

          <div className={styles.IconContainer}>
            <LayoutIcon className="h-12 mb-2 text-blue-dark" />
            <span>
              Space Planning <br />
              &amp; Layout
            </span>
          </div>
        </div>
      </Container>

      <Container className="text-center pb-8 pt-4">
        <Button variant="blue" type="link" to="/services">
          Services
        </Button>
      </Container>
    </div>
  );
};

export default Services;
