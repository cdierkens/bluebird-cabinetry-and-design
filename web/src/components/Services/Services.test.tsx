import { render } from "@testing-library/react";
import React from "react";
import Services from "./Services";

it("matches snapshot", () => {
  const { asFragment } = render(<Services />);

  expect(asFragment()).toMatchSnapshot();
});
