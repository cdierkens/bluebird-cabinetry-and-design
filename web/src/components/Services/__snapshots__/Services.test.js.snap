// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="ServicesContainer"
  >
    <div
      class="Container ServicesGrid"
    >
      <div
        class="grid grid-cols-2 sm:grid-cols-6"
      >
        <div
          class="IconContainer"
        >
          <test-file-stub
            classname="p-1 fill-current stroke-current inline-block fill-none stroke-3 h-12 mb-2 text-blue-dark"
          />
          <span>
            Cabinetry Design
            <br />
            & Sales
          </span>
        </div>
        <div
          class="IconContainer"
        >
          <test-file-stub
            classname="p-1 fill-current stroke-current inline-block fill-none stroke-3 h-12 mb-2 text-blue-dark"
          />
          <span>
            Remodeling
          </span>
        </div>
        <div
          class="IconContainer"
        >
          <test-file-stub
            classname="p-1 fill-current stroke-current inline-block fill-none stroke-3 h-12 mb-2 text-blue-dark"
          />
          <span>
            New Construction
          </span>
        </div>
        <div
          class="IconContainer"
        >
          <test-file-stub
            classname="p-1 fill-current stroke-current inline-block fill-none stroke-3 h-12 mb-2 text-blue-dark"
          />
          <span>
            Renderings
          </span>
        </div>
        <div
          class="IconContainer"
        >
          <test-file-stub
            classname="p-1 fill-current stroke-current inline-block fill-none stroke-3 h-12 mb-2 text-blue-dark"
          />
          <span>
            Material & Appliance Selections
          </span>
        </div>
        <div
          class="IconContainer"
        >
          <test-file-stub
            classname="p-1 fill-current stroke-current inline-block fill-none stroke-3 h-12 mb-2 text-blue-dark"
          />
          <span>
            Space Planning
            <br />
            & Layout
          </span>
        </div>
      </div>
    </div>
    <div
      class="Container text-center pb-8 pt-4"
    >
      <a
        class="ButtonBlue  focus:outline-none focus:ring"
        href="/services"
      >
        Services
      </a>
    </div>
  </div>
</DocumentFragment>
`;
